### PR TITLE
#1158 Feature/dashboard port in tye yaml

### PR DESCRIPTION
**Pull request by daniel-simpson**
Feature implementation for #1158 

It is a relatively small, and self-contained change with tests added, but I understand if it isn't something you want implemented in this project!  Submitting this PR to further discussion in the #1158 issue.

Thank again for everything, I've reaally enjoyed using Tye over the past six months or so.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/tye?pullRequest=1163) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/tye?pullRequest=1163) before we can accept your contribution.<br/><br/>:x: daniel-simpson [sign now](https://cla.dotnetfoundation.org/dotnet/tye?pullRequest=1163)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/tye?pullRequest=1163) it.</sub>

File: docs/reference/schema.md [hunk 1]
```diff
@@ -85,6 +85,14 @@ Allows configuring the Docker network used for `tye run`.
 
 If a network is configured, then all services running in containers will connect to the specified network. Otherwise a Docker network will be created with a generated name, and used to connect all containers.
 
+#### `dashboardPort` (int)
```
Comment (philliphoff) [inline docs/reference/schema.md:88]: Would it be possible to add this same information to the [YAML (i.e. JSON) schema](https://github.com/dotnet/tye/blob/main/src/schema/tye-schema.json) for configuration file.  That schema is used by IDEs like VS Code to provide IntelliSense to users as they edit the file.

**Review by philliphoff — APPROVED** [review]
Looks good, with just one comment re: the configuration file schema.  The additional tests are very much appreciated, too.

File: docs/reference/schema.md (hunk 1 above)
Comment (daniel-simpson) [reply docs/reference/schema.md:88]: Done!

**Comment by daniel-simpson** [comment]
I've added the requested change into the schema file.

Thanks again for the great project :D

**Pull request merged**